massa-models/Cargo.toml

```diff
@@ -18,13 +18,13 @@ config = "0.13"
 bs58 = { version = "=0.4", features = ["check"] }
 bitvec = { version = "=1.0", features = ["serde"] }
 nom = "=7.1"
+massa-proto-rs = { git = "https://github.com/massalabs/massa-proto-rs", branch = "main", features = ["tonic"] }
```

Comment:
This need to be changed to match a special revision because if new commint on main it could be problematic

Reply:
I'm agree with you, I always prefer using a rev instead of branch

Reply:
If you can make a PR to change this it would be nice !

Reply:
See #4068